src/libs/ajax/leonardo/Disks.ts

```diff
@@ -27,11 +27,6 @@ export const Disks = (signal: AbortSignal) => {
       return mapToPdTypes(disks);
     },
     disk: (project: string, name: string) => ({
-      create: (props): Promise<void> =>
```

Comment:
There is no point having this really as users should not be able to create disks in isolation in the UI correct?

Reply:
correct